.azure-pipelines/win32-conda-CI.yml

```diff
@@ -28,14 +28,21 @@ jobs:
         TENSORFLOW_PATH: tensorflow==1.15.0
         INSTALL_ORT: pip install onnxruntime==1.1.1
 
-      # UT for standalone keras need tensorflow <= 2.0.0 (python <= 3.7)
+      # UT for keras 2.3 need tensorflow <= 2.0.0
       Python37-tf200:
         python.version: '3.7'
         ONNX_PATH: onnx==1.7.0
         KERAS: keras==2.3.1
         TENSORFLOW_PATH: tensorflow==2.0.0
         INSTALL_ORT:
 
+      Python38-tf220:
+        python.version: '3.8'
+        ONNX_PATH: onnx==1.7.0
+        KERAS: keras
+        TENSORFLOW_PATH: tensorflow==2.2.0
```

Comment:
>= 2.2.0?

Reply:
yes, currently it is tensorflow 2.2.0 in pypi